LOSTANDFOUND.md

```diff
@@ -40,6 +40,14 @@ This way, their names are never removed from the RIOT repository.
 
 # Removed Features
 
+### sys/net/gnrc/application_layer/tftp [0e2a62078850e1ecc74db2db4d639cf2d8fb96d3]
+Author:
+- Sören Tempel <tempel@uni-bremen.de>
```

Comment:
`git` blame` lists `DipSwitch <dipswitch@ownage4u.nl>` as the main author of that module.

Reply:
(unless you want to be the point of contact for questions regarding that module).

Reply:
Ooooppps, no I don't. Fixed, thanks for pointing that out!